src/spyglass/utils/dj_helper_fn.py

```diff
@@ -465,3 +466,41 @@ def make_file_obj_id_unique(nwb_path: str):
         f.attrs["object_id"] = new_id
     _resolve_external_table(nwb_path, nwb_path.split("/")[-1])
     return new_id
+
+
+def populate_pass_function(value):
+    """Pass function for parallel populate.
+
+    Note: To avoid pickling errors, the table must be passed by class, NOT by instance.
```

Comment:
Did you see an error about pickling an `SSLContext` object? I circumvent that [here](https://github.com/CBroz1/spyglass/blob/ndo/src/spyglass/utils/dj_graph.py#L97-L102) in my deepcopy of the graph. That might make things simpler, letting you pass the instance

Reply:
Confirmed it's the `SSLContext` for future reference